src/extension/background-script/actions/nostr/helpers.ts

```diff
@@ -33,6 +33,24 @@ export async function addPermissionFor(method: string, host: string) {
   if (!allowance?.id) {
     return false;
   }
+
+  const findPermission = await db.permissions.get({
```

Comment:
where is this used?

Reply:
This would be used whenever the user goes back to the website after _disabling_ a permission, we have to check whether to add the permission again or just toggle enable if it already exists

Reply:
Wouldn't require this change if we proceed by deleting the permission

Reply:
I don't understand. 
we check permissions whenever a call is executed and if there is no permission the user will be prompted. 

ideally we should not add functions that we don't use. then add them when they are needed and used.

Reply:
Are these used though?: https://github.com/getAlby/lightning-browser-extension/pull/1819  
What's the reason to bring these back?

Reply:
> we check permissions whenever a call is executed and if there is no permission the user will be prompted.

Yes, we do, but here we're not deleting the permission (instead, we set it to `enabled: false`) when we disable it from the UI, which would return the enabled state. Now when we prompt the user, we just have to set it to `enabled: true` instead of adding the same permission again...

But this confusion will be removed once we delete the permissions in the first step.

I made these changes in #1819 (deleting instead of disabling as you suggested) so I'm waiting for it to be merged so that I can rebase things here to avoid confusion